scripts/gen_cyclic_test.sh

```diff
@@ -143,6 +158,31 @@ get_max_latency()
     echo $(grep "Max Latencies" "$input_file" | tr " " "\n" | sort -n | tail -1 | sed s/^0*//)
 }
 
+# Define template
```

Comment:
It is better to put all the adoc template part in the CI repository.

May be you could move this entire script in the CI repository ?